### PR TITLE
Add SLO script doc

### PR DESCRIPTION
**Pull request by kfrz**
## Background

Update documentation with a brief writeup of how the service-level screenshots were gathered.

### CODE REVIEW

**Review by alexpappasoddball — CHANGES_REQUESTED** [review]
Would like to see a little more of a instruction so that I can give this a try tomorrow and see if I can't get it to run on my machine.  Right now I'm a little lost in where to start.

File: teams/vsp/teams/tools/backend/external-service-monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+These instructions and script assume Mozilla Firefox as your browser. 
+
+This can be done with `chrome`/`chromium` and `chromedriver` but will require significant alterations.
+
+##### Prerequisites:
+
+- `ruby --version` should be `2.3.x` or greater
+- `convert --version` should be `Version ImageMagick 6.9.x` or greater [download instructions](https://imagemagick.org/script/download.ph://imagemagick.org/script/download.php)
+- `firefox --version` should be `Mozilla Firefox 72.x` or greater
```
Comment (erluti) [inline teams/vsp/teams/tools/backend/external-service-monitoring/README.md:15]: by default firefox isn't available from commandline

File: teams/vsp/teams/tools/backend/external-service-monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+- Ensure your browser profile is configured with the [SOCKS5 proxy
+config](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/scripts/socks/README.md)
+ for your machine.
+- Login to GitHub and use OAuth to login to [Grafana](http://grafana.vfs.va.gov)
+
+Now you should be prepared to execute the script.
+
+Either download script [here](./report-crawler.rb) or copy & paste the following:
+
+Find your profile name by opening `about:profiles` in Firefox - the active profile name is what you're looking for.
```
Comment (erluti) [inline teams/vsp/teams/tools/backend/external-service-monitoring/README.md:26]: make it cleare `about:profiles` is in the address bar

File: teams/vsp/teams/tools/backend/external-service-monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+Then, substitute your profile name for `$PROFILE` in the following, and run the reporting command:
+
+`PROFILE_NAME=$PROFILE ./report-crawler.rb`
```
Comment (erluti) [inline teams/vsp/teams/tools/backend/external-service-monitoring/README.md:30]: mention closing firefox

File: teams/vsp/teams/tools/backend/external-service-monitoring/report-crawler.rb [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    # TODO do the compression afterwords
+    system "convert #{tmpfile} -chop 5%x8% -gravity south -chop 0x23.5% #{tmpfile} && \
+            pngcrush -reduce -brute #{tmpfile} #{shot_filename} && \
+            rm #{tmpfile}"
+    shot_filename
+  end
+end
+spinner = TTY::Spinner.new("[:spinner] Rustlin' up a phantom 'fox ...", format: :burger)
+spinner.auto_spin
+Selenium::WebDriver::Firefox::Binary.path='/opt/firefox/firefox'
```
Comment (erluti) [inline teams/vsp/teams/tools/backend/external-service-monitoring/report-crawler.rb:65]: this path needs to be dynamic

Comment (erluti) [reply teams/vsp/teams/tools/backend/external-service-monitoring/report-crawler.rb:65]:
```
# Selenium::WebDriver::Firefox::Binary.path = '/opt/firefox/firefox'
Selenium::WebDriver::Firefox::Binary.path = '/Applications/Firefox.app/Contents/MacOS/firefox'

default_profile = Selenium::WebDriver::Firefox::Profile.from_name PROFILE_NAME
default_profile.add_extension '/Users/philip/Library/Application Support/Firefox/Profiles/gn6h0ea5.default-release/extensions/switchyomega@feliscatus.addons.mozilla.org.xpi'

options = Selenium::WebDriver::Firefox::Options.new
options.profile = default_profile
driver = Selenium::WebDriver.for :firefox, options: options


# b = Watir::Browser.new :firefox, options: { profile: PROFILE_NAME, log: { level: 'DEBUG' } }, headless: false
b = Watir::Browser.new driver, headless: false
```

File: teams/vsp/teams/tools/backend/external-service-monitoring/README.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+This can be done with `chrome`/`chromium` and `chromedriver` but will require significant alterations.
+
+##### Prerequisites:
+
+- `ruby --version` should be `2.3.x` or greater
+- `convert --version` should be `Version ImageMagick 6.9.x` or greater [download instructions](https://imagemagick.org/script/download.ph://imagemagick.org/script/download.php)
+- `firefox --version` should be `Mozilla Firefox 72.x` or greater
+- `geckodriver` [install instructions](https://github.com/mozilla/geckodriver/releases)
+- Ensure your browser profile is configured with the [SOCKS5 proxy
```
Comment (erluti) [inline teams/vsp/teams/tools/backend/external-service-monitoring/README.md:17]: mention pngcrush

File: teams/vsp/teams/tools/backend/external-service-monitoring/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+### Generate External Service Performance report
+
+These instructions explain how to use the `Selenium::WebDriver` based script to capture snapshots of each External Service dashboard.
+
+#### Prerequisites:
+
+These instructions and corresponding script assume the following:
+
+- `ruby --version` should be `2.3.x` or greater
+- `convert --version` should be `Version ImageMagick 6.9.x` or greater [install instructions](https://imagemagick.org/script/download.ph://imagemagick.org/script/download.php)
```
Comment (LindseySaari) [inline teams/vsp/teams/tools/backend/external-service-monitoring/README.md:16]: This link is malformed, looks like it needs to be: https://imagemagick.org/script/download.php

File: teams/vsp/teams/tools/backend/external-service-monitoring/README.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+- Firefox version should be `Mozilla Firefox 72.x` or greater
+- `geckodriver` [install instructions](https://github.com/mozilla/geckodriver/releases)
+
+1. Ensure your browser profile is configured to use [SOCKS proxy config](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/scripts/socks/README.md)
```
Comment (LindseySaari) [inline teams/vsp/teams/tools/backend/external-service-monitoring/README.md:20]: [May also want to link to the proxy switcher docs here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/f60352f90eaaed4ca4e8539cbf05a81bd519cb31/platform/engineering/internal-tools.md#chrome--firefox)

File: teams/vsp/teams/tools/backend/external-service-monitoring/README.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+These instructions explain how to use the `Selenium::WebDriver` based script to capture snapshots of each External Service dashboard.
+
+#### Prerequisites:
+
+These instructions and corresponding script assume the following:
+
+- `ruby --version` should be `2.3.x` or greater
+- `convert --version` should be `Version ImageMagick 6.9.x` or greater [install instructions](https://imagemagick.org/script/download.ph://imagemagick.org/script/download.php)
+- Firefox version should be `Mozilla Firefox 72.x` or greater
+- `geckodriver` [install instructions](https://github.com/mozilla/geckodriver/releases)
```
Comment (LindseySaari) [inline teams/vsp/teams/tools/backend/external-service-monitoring/README.md:18]: alternate install option: homebrew

**Review by alexpappasoddball — APPROVED** [review]
LGTM

**Comment by kfrz** [comment]
Going to merge this as is, understanding there may be some rough edges. The documentation & changelog is up to date and I verified it runs on my machine, at least.

**Pull request merged**